docs/docs/components/reanimated_swipeable.md

```diff
@@ -81,27 +81,29 @@ an argument.
 
 ### `renderLeftActions`
 
-method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
-This map describes the values to use as inputRange for extra interpolation:
+accepts a function which receives the following arguments:
 
-SharedValue: [startValue, endValue]
+- `progress` value representing swiping progress relative to the width of the returned element.  
+  Equals `0` when `swipeable` is closed, `1` when left action is open.  
+  Linearly increases to `Infinity` as left action overshoots it's open position.
+- `translation` horizontal translation of the `swipeable` in pixels.
+- `swipeable` provides an object exposing the methods listed [here](#methods)
```

Comment:
I believe this shouldn't be here in the bullet points of `renderLeftAction` at all

Reply:
are you referring to the following?
>- `swipeable` provides an object exposing the methods listed [here](#methods)

If that's the case, it **should** be, it's a reference to the `swipeable` object.

Do you think something like `swipeableReference` or `swipeableMethods` or `swipeableObject` works better?
I think standalone `swipeable` makes the most sense, but i'm not sure if it's the as verbose as the other options.

(cc @kacperkapusciak )

Reply:
Okay, now I see. I don't like `swipeable` in that case (even though it could be okay with old implementation). I'd go with either `swipeableRef` or `swipeableMethods` (preferably `swipeableMethods`)

Reply:
changed to `swipeableMethods` in [6cdf23b](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/6cdf23b6963f35202c0b65225e38e3839fe83c96)